dependency_support/org_gnu_m4/bundled.BUILD.bazel

```diff
@@ -40,66 +41,134 @@ cc_library(
     name = "m4_headers",
     textual_hdrs = glob(
         ["lib/**/*.h", "src/*.h"],
-        exclude = ["lib/spawn_int.h"],
     ) + [
         "lib/config.h",
         "lib/configmake.h",
-        "build-aux/snippet/unused-parameter.h",
+        "lib/sigsegv.h",
     ] + HEADERS_WITH_C_EXTENSION,
-    includes = ["lib", "build-aux/snippet"],
+    includes = ["lib"],
     visibility = ["//visibility:private"],
 )
 
 cc_binary(
     name = "m4",
-    srcs = glob(
-        ["lib/*.c", "src/*.c"],
-        exclude = HEADERS_WITH_C_EXTENSION + [
```

Comment:
I think these exclusions are a good thing to have. IIRC a normal m4 build (using its own build system) does not include all of those files. The m4 source distro has a lot of reimplementations of stuff that is present in any sane Linux. This is probably why you're seeing build errors.

I don't really have an opinion about how to do this though, as long as the build works.

Reply:
Oh. I switched from the "blocklist" (exclusion) to "allowlist" approach, so actually less files are included. 

that's indeed why I am seeing build errors, though. 

(if you compare the first revision abd0aac2fbb90adc269d709d4bf2c86d127b95ae with the second one eaf226bd4c3ecec8714cbb763b5f2f7d8ffe48b8, you will see that `lib/reallocarray.c` has been added)